src/operator/tensor/indexing_op.cc

```diff
@@ -360,21 +360,24 @@ NNVM_REGISTER_OP(take)
 
 This function slices the input array along a particular axis with the provided indices.
 
-Given an input array with shape ``(d0, d1, d2)`` and indices with shape ``(i0, i1)``, the output
-will have shape ``(i0, i1, d1, d2)``, computed by::
+Given an input tensor with shape ``(d0, d1, d2)`` and indices with shape ``(i0, i1)``, and axis=1,
+the output will have shape ``(d0, i0, i1, d2)``, computed by::
 
-  output[i,j,:,:] = input[indices[i,j],:,:]
+  output[:,i,j,:] = input[:,indices[i,j],:]
 
 .. note::
-   - `axis`- Only slicing along axis 0 is supported for now.
-   - `mode`- Only `clip` mode is supported for now.
+   - `axis`- Could be from -r to r-1 where r is the rank of input tensor
+   - `mode`- Could be either `clip` or `wrap`.
```

Comment:
You can move this explanation to the respective arguments and delete the note.

Reply:
Done